pom.xml

```diff
@@ -698,6 +699,12 @@
                 <version>${thrift.libthrift.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>com.github.tomakehurst</groupId>
+                <artifactId>wiremock-jre8</artifactId>
+                <version>${wiremock.version}</version>
```

Comment:
should have the `test` scope, and moved close to the other test library declarations